### ISSUE TITLE
Question about regression model

### ISSUE THREAD

**Issue by TheTalos** [tyiannak/pyAudioAnalysis#4]
Hi!
I am trying to create a regression model based on some musical data I have, using pyAudioAnalysis. My training data contains some WAV audio clips and two csv files: one for arousal and one for valence, with values between 1 and 9 (but my data does not include neither end values). What kind of normalization should I apply to get an output with values between -1 and 1?
Does the duration of the audio files in the test set necessarily need to be similar or equal to the duration of the ones in the training set?

**Comment by tyiannak** [comment]
Hi.
Also, you don't have to normalize in the 1 -1 range. However, if you want
it you need to do such a normalization then just do (x - (Max-Min)/2) /
(Max - Min)
The length of the test/train audio files need not be the same.
